src/applications/personalization/rated-disabilities/actions/index.js

```diff
@@ -0,0 +1,27 @@
+import { getData } from '../util';
+
+export const FETCH_RATED_DISABILITIES_SUCCESS =
+  'FETCH_RATED_DISABILITIES_SUCCESS';
+export const FETCH_RATED_DISABILITIES_FAILED =
+  'FETCH_RATED_DISABILITIES_FAILED';
+
+export function fetchRatedDisabilities() {
+  return async dispatch => {
+    const response = await getData(
+      '/disability_compensation_form/rated_disabilities',
+    );
+
+    if (response.errors) {
```

Comment:
Since an empty array is considered truthy, we might want to use something like Lodash's `isEmpty` method here to ensure we don't accidentally throw an error state when there are no errors.

```
import { isEmpty } from 'lodash';

...

if (!isEmpty(response.errors)) {
```

Reply:
As I see what you're saying. I'm wondering if `isEmpty` is necessary though, since it seems like `errors` doesn't get sent back to the frontend at all on a successful API call. If an error is returned from the API, then it looks like `errors` is the only thing sent back to the frontend. 

https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/form_526/getRatedDisabilities

Reply:
Currently it doesn't send back `errors` if there are no errors, but I just mentioned this since the API can change in the future 🙂